common/source/docs/common-polygon_fence.rst

```diff
@@ -12,6 +12,8 @@ ArduPilot includes support for polygon fences with up to 70 points and for pure
 
 This feature is an extension of the simpler home-based :ref:`cylindrical fence <common-ac2_simple_geofence>` and can be combined with it.
 
+.. warning:: While Copter and Rover can use :ref:`common-object-avoidance-landing-page` strategies to avoid most fence breaches, Plane only has airborne aircraft :ref:`ADSB avoidance <common-ads-b-receiver>`, for safety plan your polygon fences anticipating the path of whatever :ref:`FENCE_ACTION<FENCE_ACTION>` is selected upon breach. This usually means significantly larger fence boundaries than what is being protected.
```

Comment:
```suggestion
.. warning:: While Copter and Rover can use :ref:`common-object-avoidance-landing-page` strategies to avoid most fence breaches, Plane only has airborne aircraft :ref:`ADSB avoidance <common-ads-b-receiver>`. For safety, plan your polygon fences anticipating the path of whatever :ref:`FENCE_ACTION<FENCE_ACTION>` is selected upon breach. This usually means significantly larger fence boundaries than what is being protected.
```